modules/check_in/app/workers/check_in/travel_claim_submission_worker.rb

```diff
@@ -5,9 +5,67 @@
 module CheckIn
   class TravelClaimSubmissionWorker
     include Sidekiq::Worker
-    include Sidekiq::MonitoredWorker
+    include SentryLogging
+
     sidekiq_options retry: false
 
-    def perform(uuid, appointment_date); end
+    SUCCESS_TEMPLATE_ID = Settings.vanotify.services.check_in.template_id.claim_submission_success_text
+    DUPLICATE_TEMPLATE_ID = Settings.vanotify.services.check_in.template_id.claim_submission_duplicate_text
+    ERROR_TEMPLATE_ID = Settings.vanotify.services.check_in.template_id.claim_submission_error_text
+
+    SMS_SENDER_ID = Settings.vanotify.services.check_in.sms_sender_id
+
+    STATSD_ERROR_NAME = 'worker.checkin.travel_claim.error'
+    STATSD_SUCCESS_NAME = 'worker.checkin.travel_claim.success'
+
+    def perform(uuid, appointment_date)
+      response = TravelClaim::Response
+      check_in_session = CheckIn::V2::Session.build(data: { uuid: })
+      mobile_phone = TravelClaim::RedisClient.build.mobile_phone(uuid:)
+
+      logger.info("Submitting travel claim for #{uuid}, #{appointment_date}")
+      claims_resp = TravelClaim::Service.build(
+        check_in: check_in_session,
+        params: { appointment_date: }
+      ).submit_claim
+
+      claim_number = claims_resp.dig(:data, 'value', 'claimNumber')
+      template_id = case claims_resp.dig(:data, :code)
+                    when response::CODE_SUCCESS
+                      SUCCESS_TEMPLATE_ID
+                    when response::CODE_CLAIM_EXISTS
+                      DUPLICATE_TEMPLATE_ID
+                    else
+                      ERROR_TEMPLATE_ID
+                    end
+      send_notification(mobile_phone:, appointment_date:, template_id:, claim_number:)
+      StatsD.increment(STATSD_SUCCESS_NAME)
+    end
+
+    def send_notification(opts = {})
+      notify_client = VaNotify::Service.new(Settings.vanotify.services.check_in.api_key)
+
+      logger.info("Sending notification to: #{opts[:mobile_phone]}, using template_id: #{opts[:template_id]}")
+      notify_client.send_sms(
+        phone_number: opts[:mobile_phone],
+        template_id: opts[:template_id],
+        sms_sender_id: SMS_SENDER_ID,
+        personalisation: {
+          claim_number: opts[:claim_number],
+          appointment_date: opts[:appointment_date]
```

Comment:
Wondering if the API call will succeed if the personalization object is sent for duplicate & error template ids?

Reply:
I think it will, as the current template doesn't include any personalization and I've been testing it locally with the code above and it's been working.

Reply:
validated that the sms is sent correctly if we pass personalization attributes that are not used in the templates. resolving this comment.